docs/HTTPClient.md

```diff
@@ -27,6 +27,8 @@ Both proxies expose the methods to perform a specific HTTP request to service.
  * `patch(path, body, querystring, options)`
  * `delete(path, body, querystring, options)`
 
+All methods return a *Promise object*. You can access to response body trough the `payload` property of the returned object.
```

Comment:
If we write how to access to the body, we should also explain how to access to other fields of the response such as headers. What do you think?

Reply:
Yes, we should, Can we access to headers with `payload.headers`?

Reply:
Yes, headers and statusCode

Reply:
@davidebianchi I have updated the file